src/website/views/components/dropdown/_examples.pug

```diff
@@ -775,6 +775,100 @@ p.-text
         <!-- Include your custom logic here to retain dropdown selection -->
       </script>
 
+.-portal--hide
+  h3 Icon
+  p.-text
+    | There is available property <code>icon</code> which allows to add an icon as the dropdown trigger.
```

Comment:
Hi! Just some grammar stuff:

`There is an available property, <code>icon</code>, which allows adding an icon as the dropdown trigger.`